rhizome/host/lib/vm_setup.rb

```diff
@@ -437,7 +437,7 @@ def cloudinit(unix_user, public_keys, gua, nics, swap_size_bytes, boot_image, dn
     interfaces = nics.map { "interface=#{_1.tap}" }.join("\n")
     dnsmasq_address_ip6 = NetAddr::IPv6.parse("fd00:0b1c:100d:53::")
     address_mapping = boot_image.include?("github") ?
-      "address=/localhost.blob.core.windows.net/#{nics.first.net4.split("/").first}" :
+      "address=/ubicloudhostplaceholder.blob.core.windows.net/#{nics.first.net4.split("/").first}" :
```

Comment:
maybe `ubicloudlocal`

Reply:
Keeping it as it is to not face any issues further in case "local" is checked somehow in future